packages/admin/src/cli.ts

```diff
@@ -48,6 +48,11 @@ const COMMON_COMMAND_ARGUMENTS = {
       describe: 'Derivation path to be used for deriving the wallet account',
     },
   },
+  airnodeMnemonic: {
```

Comment:
You could use `mnemonicCommands` instead of adding a new one since `derivation-path` is optional.

Reply:
I'm just trying to re-use the same object and also make it simpler to the end user. I mean, if I'm used to adding `--mnemonic` arg to all other commands that require a mnemonic it might be simpler to keep the same name instead of having a single command that requires a `--airnode-mnemonic`.

Reply:
I'm not sure TBH because I don't think this tool has been used enough to determine what is simpler for the user but I do see we are not being consistent if we make a "special" mnemonic arg just for this command. For instance: `sponsor-requester`, `unsponsor-requester` and `request-withdrawal` should only be called with the sponsor wallet mnemonic. Does that mean we need to use a `--sponsor-mnemonic` arg instead?
I feel like the arg name should be just `--mnemonic` and if that mnemonic should be for a specific wallet then those are details that must be added in the docs for the user to know, don't you think?

Reply:
This is a good point. I forgot that we use this for other commands in a similar way.

> I feel like the arg name should be just --mnemonic and if that mnemonic should be for a specific wallet then those are details that must be added in the docs for the user to know, don't you think?

I think user should not need to go to docs for this and it should be clear even from the CLI. Especially, since user can easily confuse these concepts when first trying airnode... I think we should bring this up at the planning call.

For now, I am going to use the mnemonic for consistency.